studio-docs/source/metrics/field-usage.mdx

```diff
@@ -181,25 +183,47 @@ To help resolve these references, the Reviews subgraph _must_ return each `Produ
-In order to report referencing operation statistics, your server must be running Apollo Server 3.6 or newer. If your graph is federated, there are no specific requirements for your subgraphs in order to support referencing operation statistics.
+#### Field executions
 
-There are no specific requirements in order to report field execution statistics from a non-federated Apollo Server. To report field execution statistics from an Apollo Gateway, your subgraphs must support federated tracing; these are the server libraries listed with "FTV1" in [the subgraph-compatible library list](https://www.apollographql.com/docs/federation/other-servers/). If some of your subgraphs support federated tracing and others do not, field executions in subgraphs that do not support federated tracing will not contribute to statistics.
+To report metrics for field executions, your GraphQL server can run any recent version of Apollo Server 2.x or 3.x.
 
-Calculating field execution statistics can potentially have performance impacts on large queries and high-traffic graphs. This is especially the case for federated graphs, because field execution statistics are currently transmitted from subgraphs to federated graphs in the form of detailed execution traces inside the body of the response from the subgraph to the Gateway. Starting with Apollo Server 3.6, you can disable field-level instrumentation for some or all operations via the [`fieldLevelInstrumentation` option to `ApolloServerPluginUsageReporting`](https://apollographql.com/docs/apollo-server/api/plugin/usage-reporting/#fieldlevelinstrumentation).
+If you have a [federated graph](https://apollographql.com/docs/federation), your subgraphs must support federated tracing. For compatible libraries, see the **FTV1** column of [this table](https://www.apollographql.com/docs/federation/other-servers/).
 
-Disabling field-level instrumentation for a particular request has three effects:
+> If _some_ of your subgraphs support federated tracing and others _don't_, only field executions in compatible subgraphs are reported to Apollo.
 
-- The request does not contribute to the "field executions" statistic on the Fields page.
-- The request does not contribute to field-level execution timing hints that can be optionally displayed in Studio Explorer and in vscode-graphql.
-- The request does not produce a trace that can be viewed on the Traces section of the Operations page.
+### Performance considerations
 
-These requests _will_ still contribute to most features of Studio, such as schema checks, the Operations page, and the "referencing operations" statistic on the Fields page.
+Calculating field execution metrics can affect performance for large queries or high-traffic graphs. This is especially true for [federated graphs](https://apollographql.com/docs/federation), because a subgraph calculates each incoming operation's field execution metrics _before_ responding to the gateway (the metrics are included in the response).
```

Comment:
Hmm, not exactly this — it's not that it calculates it first but that it includes it as part of the response.